### PR TITLE
Update 22-portal-content.sql

### PR DESCRIPTION
**Pull request by jjeroch**
- [x] - updated App URL
- [x] - deleted 4 incorrect roles for a marketplace app
- [x] - added for an marketplace app new roles
- [x] - added app_subscription_status value for company_assigned_apps
- [x] - add additional apps released lately

### CODE REVIEW

**Review by Phil91 — APPROVED** [review]
Looks fine by me

**Comment by jjeroch** [comment]
Changes closed/added. Review can get finalized to merge the pr

File: postgres/init/22-portal-content.sql [hunk 1]
```diff
@@ -115,6 +118,29 @@ COPY portal.app_detail_images (id, app_id, image_url) FROM stdin;
 5cf74ef8-a0a1-0000-a000-000000000007	ac1cf001-7fbc-1f2f-817f-bce05744000b	CE_Marketplace_Image1.png
 5cf74ef8-a0a1-0000-a000-000000000008	ac1cf001-7fbc-1f2f-817f-bce05744000b	CE_Marketplace_Image2.png
 5cf74ef8-a0a1-0000-a000-000000000009	ac1cf001-7fbc-1f2f-817f-bce05744000b	CE_Marketplace_Image3.png
+5cf74ef8-a0a1-0000-a000-0000000000010	5cf74ef8-e0b7-4984-a872-474828beb5d5	Smart Factory-Details1.png
```
Comment (ntruchsess) [inline postgres/init/22-portal-content.sql:121]: 5cf74ef8-a0a1-0000-a000-0000000000010 is not a valid UUID (it's to long)
(same applies to the ids below)

**Review by ntruchsess — CHANGES_REQUESTED** [review]

File: postgres/init/22-portal-content.sql (hunk 1 above)
Comment (jjeroch) [reply postgres/init/22-portal-content.sql]: fixed

File: postgres/init/22-portal-content.sql [hunk 2]
```diff
@@ -161,6 +196,16 @@ ac1cf001-7fbc-1f2f-817f-bce05744000b	en
 ac1cf001-7fbc-1f2f-817f-bce0573f0009	en
 5cf74ef8-e0b7-4984-a872-474828beb5d4	en
 5cf74ef8-e0b7-4984-a872-474828beb5d4	de
+5cf74ef8-e0b7-4984-a872-474828beb5d5	en
+5cf74ef8-e0b7-4984-a872-474828beb5d6	en
+5cf74ef8-e0b7-4984-a872-474828beb5d7	en
+5cf74ef8-e0b7-4984-a872-474828beb5d7	de
+5cf74ef8-e0b7-4984-a872-474828beb5d7	pt
```
Comment (ntruchsess) [inline postgres/init/22-portal-content.sql:203]: 'pt' is not a valid language-shortname. Currently we only have 'de' and 'en'.
you need to also update 21-portal-basedata.sql:
COPY portal.languages (short_name, long_name_de, long_name_en) FROM stdin;
de	deutsch	german
en	englisch	english
\.

**Review by ntruchsess — CHANGES_REQUESTED** [review]

File: postgres/init/22-portal-content.sql (hunk 2 above)
Comment (jjeroch) [reply postgres/init/22-portal-content.sql:203]: true, I took the wrong table. I have validated against the country table. With the new 20file structure, a DB Table connection picture will get more and more relevant I believe

Comment (ntruchsess) [reply postgres/init/22-portal-content.sql:203]: If 'language short name' is meant to be the ISO 639-1 language code (https://de.wikipedia.org/wiki/Liste_der_ISO-639-1-Codes) - (which I strongly suggest) than I agree - we should have s separate table that contains the languages where translations of portal-resources actually do exist for (the list of language-codes (which is predefined in ISO-639-1 may have more entries).

File: postgres/init/22-portal-content.sql [hunk 3]
```diff
@@ -398,6 +467,20 @@ This is only a test text	This is only a test text	ac1cf001-7fbc-1f2f-817f-bce057
...
+The Smart Factory Web Portal is a web-based application to allow suppliers and MaaS platforms to offer production capabilities and capacities. At the same time, the Smart Factory Web Portal offers a search environment where a buyer can upload engineering files and configure procurement requirements which the Smart Factory Web Search Engine the portal exploits uses to calculate all possible supply chain alternatives by matching it to the registered supplier information.
```
Comment (ntruchsess) [inline postgres/init/22-portal-content.sql:470]: here column-data for 'description-short', app_id and language_short_name are missing

**Review by ntruchsess — CHANGES_REQUESTED** [review]

**Review by ntruchsess — APPROVED** [review]

**Pull request merged**